prj.conf

```diff
@@ -35,7 +34,6 @@ CONFIG_ESPI_FLASH_CHANNEL=y
 # SAF required drivers
 CONFIG_SPI=y
 CONFIG_ESPI_SAF=y
-CONFIG_SAF_SPI_CAPACITY=32
```

Comment:
CONFIG_SAF_SPI_CAPACITY=32 can be retained here as that is the default configuration for all intel platforms with onboard EC currently. CONFIG_SAF_SPI_CAPACITY=16 can be redefined only for mec1501modular_assy6885.conf.